Nemt/nemt.h

```diff
@@ -5,10 +5,15 @@
 #include <Pixy.h>
 
 #define CAMERA_CENTER 160
+#define NOT_FOUND_NEGATIVE (-404)
+#define NOT_FOUND_POSITIVE 404
```

Comment:
Why do we repeat `404` twice? Wouldn't it be better to just define `NOT_FOUND 404` and then use `NOT_FOUND` and `-(NOT_FOUND)` respectively?